dotfiles.sh

```diff
@@ -14,9 +14,9 @@ srcdir="@datadir@"
 #   command-line argument parsing
 usage () {
     echo "dotfiles: USAGE: dotfiles [-v|--version] [-q|--quiet] [-f|--force] [-p|--preserve] [-r|--remote <user>@<host>] <home-directory>" 1>&2
+    echo "                 dotfiles -e [<file>...]" 1>&2
 }
 if [ $# -eq 0 ]; then
-    echo "dotfiles: ERROR: invalid number of arguments" 1>&2
```

Comment:
Why are you removing this line? It seems to be not unrelated to the new `-e` options.

Reply:
This is true but IMHO shell tools are expected to emit their usage either when called without arguments or when called with an explicit –h or –help option. One of those should work and is not an error. It is just the user asking for advice how to use the tool. YMMV.